cypress/page/login.page.ts

```diff
@@ -0,0 +1,29 @@
+class LoginPage {
+  private emailSelector: string;
+  private emailUser: string;
+  private passwordSelector: string;
+  private passwordUser: string;
+  private signIn: string;
+
+  constructor () {
+    this.emailSelector = "#email";
+    this.emailUser = "aperdomobo@gmail.com";
+    this.passwordSelector = "#passwd";
+    this.passwordUser = "WorkshopProtractor";
+    this.signIn = "#SubmitLogin > span";
+  }
+
+  public typeEmail(): void{
+    cy.get(this.emailSelector).type(this.emailUser)
+  }
+
+  public typePassword(): void {
+    cy.get(this.passwordSelector).type(this.passwordUser)
+  }
+
+  public goToAddress(): void {
+    cy.get(this.signIn).click()
+  }
```

Comment:
you can simplify this just using one function login with those 3 steps (type email, type pwd and click in sign in)